test/multiverse/lib/multiverse/suite.rb

```diff
@@ -681,6 +677,16 @@ def verbose?
     def exclude?(file)
       EXCLUDED_FILES.include?(File.basename(file))
     end
+
+    def execution_message
+      label = should_serialize? ? 'serial' : 'parallel'
+      env_count = filter_env ? 1 : environments.size
+      env_plural = env_count > 1 ? 'environments' : 'environment'
+      opening = "\nRunning \"#{suite}\" suite in"
+      body = environments.instrumentation_permutations.map { |p| "#{env_count} #{p.upcase} #{env_plural}" }
+      ending = "in #{label}"
+      message = [opening, body.join(' and '), ending].join(' ')
```

Comment:
You're about 1 byte away from me recommending that we introduce the [new HAML](https://github.com/haml/haml/releases/tag/v6.0.0.beta.1) for this much formatting! 😆